test/data/mnt/boot/system-proxy/redsocks.conf

```diff
@@ -10,8 +10,8 @@ redsocks {
     local_ip = 127.0.0.1;
     local_port = 12345;
     ip = example.org;
-	port = 1080;
+    port = 1080;
     type = socks5;
     login = "foo";
-	password = "bar";
+    password = "bar";
```

Comment:
I would have probably just push-force to my PR branch if I missed some validation, but I understand it may not feel very comfortable

Reply:
Will do! I will say, I've had nightmares of accidentally force-pushing to production 😰

Reply:
Squashed into 12ce1fd2c23f606040afcc27f2b1e4c5dfd48590